first attempt.cpp

```diff
@@ -0,0 +1,123 @@
+#include<iostream>
+#include<math.h>
+#include<string>
+#include<vector>
+
+using namespace std;
+enum type
+{
+    SHIRT,
+    JEANS,
```

Comment:
Variables should be lowercase

Reply:
it is not changed or responded, but PR is approved